agents/community/literature-review/src/literature_review/server.py

```diff
@@ -26,39 +81,30 @@ def encode_value(value):
     def serialize_json(value):
         try:
             return json.dumps(value, indent=2, default=encode_value)
-        except:
+        except (TypeError, ValueError):
             return ""
 
     try:
         async for value in team.run_stream(
-            task=input.text,
+            task=inputs[-1].__str__(),
         ):
             if isinstance(value, BaseChatMessage) or isinstance(value, BaseAgentEvent):
                 action = {
                     "content": value.content,
                     "type": value.type,
                     "source": value.source,
                 }
-                delta = TextOutput(
-                    text="",
-                    logs=[
-                        None,
-                        Log(
-                            message=serialize_json(action),
-                            **action,
-                        ),
-                    ],
-                )
-                yield delta
+                
+                yield MessagePart(content=serialize_json(action))
```

Comment:
Let's yield these as plain dict, this shouldn't be part of the output
```suggestion
                yield {"message": serialize_json(action)}
```